content/guides/composite_monitors.md

```diff
@@ -0,0 +1,161 @@
+
+![create-composite-3](/static/images/composite_monitors/create-composite-3.png)
+
+### Set a trigger condition
+
+In the 'trigger when' field, write your desired trigger condition using boolean operators, referring to individual monitors by their labels in the form (a, b, c, etc). You can use parentheses to control operator precedence and create more complex conditions. 
+
+These are all valid trigger conditions: 
+ 
+~~~
+!(a && b)
+a || b && !c
+(a || b) && (c || d)
+~~~
+
+Outside of a composite monitor's New Monitor and Edit forms (e.g. on its Status page), its individual monitors are known by their numeric IDs:
+
+![composite-status](/static/images/composite_monitors/composite-status.png)
+
+In the API, a composite monitor's trigger condition is called its query. While a non-composite monitor's query can encapsulate many things — a metric, tags, an `avg` function, a group-by, etc — a composite monitor's query is simply its trigger condition, defined in terms of its constituent monitors.
+
+For two non-composite monitors with the following queries:
+
+~~~
+"avg(last_1m):avg:system.mem.free{role:database} < 2147483648" # monitor ID: 1234
+"avg(last_1m):avg:system.cpu.system{role:database} > 50" # monitor ID: 5678
+~~~
+
+a composite monitor's query is simply `"1234 && 5678"`, `"!1234 || 5678"`, etc.
+
+
+### Write a notification message
+
+Write a notification message as you would with any other monitor, using the @-syntax (e.g. @you@example.com) to notify individuals or teams. In addition to your own message, notifications for the composite monitor will show the status of the individual monitors:
+
+~~~
+[Triggered] AAA Steve Test Composite 1
+
+Steve, your composite monitor has triggered! @steve@example.com
+
+Query: 51253 || 51719
+
+1 Alert | 3 OK
+
+* AAA Steve Test 1
+  ID: 51253
+  2 availability-zone groups
+  1 Alert | 3 OK
+
+* AAA Steve Test 2
+  ID: 51719
+  1 availability-zone group
+  4 OK
+
+The monitor was last triggered at Fri Mar 24 2017 15:56:21 EDT (28 secs ago)
+~~~
+
+After setting any other miscellaneous options, click 'Save'.
+
+## How composite monitors work
+
+In this section, we look at a few examples to show **how** and **when** Datadog will alert on composite monitors.
+
+### How to alert?
+
+Consider a composite monitor that uses three individual monitors — A, B, and C — and a trigger condition `A && B && C`. How many simultaneous alerts might you potentially receive from the composite monitor? That depends on its individual monitors' alert types. 
+
+Let's look at three examples with varying individual monitors. We'll consider how the composite monitor behaves in each case.
+
+#### All simple alert monitors
+
+The composite monitor will also have a simple alert type, i.e. it will only send up to one alert per evaluation cycle. The composite monitor triggers when the queries for A, B, and C are all true at the same time.
```

Comment:
I thought about that too, but I wasn't to avoid just restating the example title. I thought the lead-in sentence explained what the examples mean:

"Let's look at three examples with varying individual monitors. We'll consider how the composite monitor behaves in each case."

If it doesn't explain it well enough, maybe I'll re-word the lead-in?